src/main/java/by/itech/lab/supplier/dto/mapper/UserMapper.java

```diff
@@ -30,6 +31,7 @@ public UserDto map(User user) {
                 .deletedAt(user.getDeletedAt())
                 .addressDto(user.getAddress() != null ? addressMapper.map(user.getAddress()) : null)
                 .warehouseDto(Objects.isNull(user.getWarehouse()) ? null : warehouseMapper.map(user.getWarehouse()))
+                .customerDto(customerMapper.map(user.getCustomer()))
```

Comment:
We need to handle null case like in the line above.

Reply:
done